docs/UserGuide.md

```diff
@@ -28,9 +59,22 @@ management tasks done faster than traditional GUI apps.
 
 1. Refer to the [Features](#features) below for details of each command.
```

Comment:
Will this hyperlink still work after u added '3.' for 'features' below?

Reply:
I have updated the features to contain the anchor tag. Thanks for pointing it out!